cmake/Modules/FindLibTorrentRasterbar.cmake

```diff
@@ -0,0 +1,59 @@
+# - Try to find libtorrent-rasterbar
```

Comment:
this whole file still has lots of references to libtorrent. There is no dependency on libtorrent, right?